### PR TITLE
Fix: Generate 3.6 Gateway kong.conf reference

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Regenerating the configuration reference for Gateway based on latest 3.6.1.1 patch. 
We didn't generate it with the release and there were significant changes to the file in that version. 
Now that these files are split out by version, this is easier to see. 

Also making some tweaks to run.lua and data.lua to use the same text that we have in the page description, and to remove a badge that doesn't need to exist.

Fixes https://konghq.atlassian.net/browse/DOCU-2889.

Will do the same for 3.5 with upcoming patch release (should be end of next week).

### Testing instructions

Preview link: https://deploy-preview-7115--kongdocs.netlify.app/gateway/latest/reference/configuration/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

File: app/_src/gateway/reference/configuration/configuration-3.6.x.md [hunk 1]
```diff
@@ -972,6 +972,8 @@ Defines the TLS ciphers served by Nginx.
 
 Accepted values are `modern`, `intermediate`, `old`, `fips` or `custom`.
 
+If you want to enable TLSv1.1, this value has to be `old`.
```
Comment (cloudjumpercat) [inline app/_src/gateway/reference/configuration/configuration-3.6.x.md:975]:
```suggestion
If you want to enable TLSv1.1, this value must be `old`.
```

File: app/_src/gateway/reference/configuration/configuration-3.6.x.md [hunk 2]
```diff
@@ -997,9 +999,11 @@ Enables the specified protocols for client-side connections. The set of
 supported protocol versions also depends on the version of OpenSSL Kong was
 built with. This value is ignored if `ssl_cipher_suite` is not `custom`.
 
+If you want to enable TLSv1.1, you should set `ssl_cipher_suite` to `old`.
```
Comment (cloudjumpercat) [inline app/_src/gateway/reference/configuration/configuration-3.6.x.md:1002]:
```suggestion
If you want to enable TLSv1.1, you must set `ssl_cipher_suite` to `old`.
```
Maybe? Just to match the previous wording where it was a requirement.

**Review by cloudjumpercat — APPROVED** [review]
Left a few small suggestions, but everything LGTM!

File: app/_src/gateway/reference/configuration/configuration-3.6.x.md (hunk 1 above)
Comment (lena-larionova) [reply app/_src/gateway/reference/configuration/configuration-3.6.x.md:975]: I have to ignore these unfortunately, since this is a generated file :(

**Pull request merged**